### ISSUE TITLE
Query builder doesn't place USING section last on INSERT

### ISSUE THREAD

**Issue by crispywalrus** [websudos/phantom#378]
The query here:

````
  "should append a ttl after a lightweight clause" in {
        val query = Recipes.insert.ifNotExists().value(_.url, "test").value(_.ingredients, List("test")).ttl(1000).queryString
      
        query shouldEqual "INSERT INTO phantom.Recipes (url, ingredients) VALUES('test', ['test']) IF NOT EXISTS USING TTL 1000"
      }
````

produces the cql ````INSERT INTO phantom.Recipes (url, ingredients) VALUES('test', ['test']) USING TTL 1000 IF NOT EXISTS```` This statement generates a syntax error on execution. The CQL should be ````INSERT INTO phantom.Recipes (url, ingredients) VALUES('test', ['test']) IF NOT EXISTS USING TTL 1000```` This seems to be due to a deliberate change to order the lightweight transaction clause last but actually the USING (both TTL and TIMESTAMP) needs to come as the last section.

**Comment by crispywalrus** [comment]
To be clear, this is the documented syntax in all versions of CQL that support lightweight transactions. See here https://cassandra.apache.org/doc/cql3/CQL.html#insertStmt

**Comment by crispywalrus** [comment]
Fixed with #379
